app/utils/i18n.ts

```diff
@@ -1,9 +1,9 @@
 export default {
   // This is the list of languages your application supports
-  supportedLngs: ["fi"],
+  supportedLngs: ["fi", "en"],
```

Comment:
We don't have English translations, now do we?

Reply:
I'm not sure if it's needed by i18next, but we should not specify any languages that we don't have translations for.

Reply:
good point, it needs to be removed. There was an error with the finnish translations not working, adding it when the fallback language was still english, made an error disappear, but it should have been removed when fallback languages was switched to finnish.

Reply:
english removed